ex5/ex5.c

```diff
@@ -15,7 +15,29 @@ char* msg3 = "hello world #3";
 
 int main()
 {
-    // Your code here
+    int pipe1[2];
```

Comment:
We don't need two arrays to hold the pipe handles here. We actually just need one array to hold one write handle and one read handle. These will be shared between both the parent and child processes. One process will write to the write handle and the other process will read from the read handle.